deployment/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM eclipse-temurin:17.0.2_8-jdk-focal
+COPY ./gradlecache /root/.gradle
```

Comment:
- Dieses File habe ich in Dala-50 gelöscht.
- Dala-50 wurde in Main gemergt
- Main wurde hier rein gemergt
- Wie kann es sein, dass das File wieder da ist??  Sehr ominös

Reply:
wurde gelöscht